great_expectations/data_context/data_context.py

```diff
@@ -2764,6 +2764,11 @@ def add_checkpoint(
         # Next two fields are for LegacyCheckpoint configuration
         validation_operator_name: Optional[str] = None,
         batches: Optional[List[dict]] = None,
+        # the following are arguments used by SimpleCheckpoint
```

Comment:
@roblim Please add the number of SimpleCheckpoint-related variables to this comment just in case somebody later adds a non-related variable below them.  Thanks!